### DISCUSSION TITLE
Spoorn Armor Attributes模组该如何刷新装备属性 Refreshing armor modifiers

### DISCUSSION THREAD

**Discussion by chenxc468** [Rad586/Forget-Me-Not#71] [Q&A] [answered]
模组里说可以使用钻石和青金石在铁砧内升级和刷新装备词条，但我无法在铁砧里使用这些，是有在配置文件里修改所需物品吗？

**Comment by Rad586** [comment, marked as answer]
是。现在是铁块刷新，金块升级。游戏内可以在铁砧界面左上角看提示。
You can use iron blocks to refresh the modifers, and gold blocks to upgrade them. You can find a lovely tip on the upper-left edge of anvil gui in game.
![image](https://github.com/Rad586/Forget-Me-Not/assets/99306685/636285b7-4938-40b0-a92c-60a27f1e8c90)
